src/Assignments/ProblemIframe.tsx

```diff
@@ -346,7 +346,7 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
                     }
                 }}
                 title='Problem Frame'
-                style={{width: '100%', height: height, border: 'none', minHeight: '350px', visibility: (loading || error) ? 'hidden' : 'visible'}}
+                style={{width: '100%', height: height, border: 'none', minHeight: readonly ? '' : '350px', visibility: (loading || error) ? 'hidden' : 'visible'}}
```

Comment:
Shouldn't it be `undefiend : 350px`
I've definitely weird issues with this before, I think if that didn't work you could _.omitBy(styles, _.isUndefined)
The problem with this (which is not an issue now) is I would think you would overwrite it with an invalid style, so if something else styled it you would lose that styling when you just didn't want it applied